pkg/util/github/github.go

```diff
@@ -125,7 +125,7 @@ func DownloadOpenShiftReleaseBinary(binaryType OpenShiftBinaryType, osType minis
 		return errors.Wrap(err, fmt.Sprintf("Cannot download OpenShift release asset %d", assetID))
 	}
 	if len(url) > 0 {
-		glog.V(2).Infof("Downloading %s %s\n", binaryType.String(), *release.TagName)
+		fmt.Println(fmt.Sprintf("Downloading OpenShift client binary %s with version '%s'", binaryType.String(), *release.TagName))
```

Comment:
@hferentschik So there are two types of binary i.e. OpenShift server or client. I do not think we will ever going to download the OpenShift server binary in this function unless we are moving away from "cluster up". Which I think not going to happen in foreseeable future. So saying `OpenShift client binary` should be fine IMO.